op-batcher/batcher/sync_actions.go

```diff
@@ -27,7 +27,7 @@ type syncActions struct {
 
 func (s syncActions) String() string {
 	return fmt.Sprintf(
-		"SyncActions{blocksToPrune: %d, channelsToPrune: %d, clearState: %v, blocksToLoad: [%d, %d]}", s.blocksToPrune, s.channelsToPrune, s.clearState, s.blocksToLoad.start, s.blocksToLoad.end)
+		"SyncActions{blocksToPrune: %d, channelsToPrune: %d, clearState: %v, blocksToLoad: [%v]}", s.blocksToPrune, s.channelsToPrune, s.clearState, s.blocksToLoad)
```

Comment:
```suggestion
		"SyncActions{blocksToPrune: %d, channelsToPrune: %d, clearState: %v, blocksToLoad: %v}", s.blocksToPrune, s.channelsToPrune, s.clearState, s.blocksToLoad)
```
otherwise you'll have `[...]` around the struct print